src/map/enmity_container.cpp

```diff
@@ -171,6 +174,7 @@ void CEnmityContainer::UpdateEnmity(CBattleEntity* PEntity, int32 CE, int32 VE,
         VE = std::clamp((int32)(VE * bonus), 0, EnmityCap);
 
         m_EnmityList.emplace(PEntity->id, EnmityObject_t {PEntity, CE, VE, true, maxTH});
+		m_EnmityList.emplace(PEntity->id, EnmityObject_t {PEntity, CE, VE, true, maxCR});																		   
```

Comment:
don't add enmity twice (that's essentially what this does, instead what you can do since you have expanded that struct

replace the line above this one as

`m_EnmityList.emplace(PEntity->id, EnmityObject_t {PEntity, CE, VE, true, maxTH, maxCR});`